lib/Controller/Schedule.php

```diff
@@ -2292,8 +2292,18 @@ public function grid(Request $request, Response $response)
 
                 if ($event->recurrenceType === 'Week') {
                     $weekdays = Carbon::getDays();
-
-                    $repeatsOn = $weekdays[$event->recurrenceRepeatsOn - 1];
+                    $repeatDays = explode(',', $event->recurrenceRepeatsOn);
+                    $i = 0;
+                    foreach ($repeatDays as $repeatDay) {
+                        // Carbon getDays starts with Sunday,
+                        // return first element from that array if in our array we have 7 (Sunday)
+                        $repeatDay = ($repeatDay == 7) ? 0 : $repeatDay;
+                        $repeatsOn .= $weekdays[$repeatDay];
+                        if ($i < count($repeatDays) - 1) {
+                            $repeatsOn .= ' and ';
```

Comment:
I don't think we can have the English word "and" in here - would a comma or & be better?

Reply:
sure, updated in the commit below.